Agents/BMSUpdateAgent/Dockerfile

```diff
@@ -23,7 +23,7 @@ FROM tomcat:9.0 as agent
 
 WORKDIR /app
 
```

Comment:
Should add ENV in case people build with `docker build`

Reply:
The ENV is included in the stack json file. Since the agent is designed to work in the stack, they should modify the stack json file accordingly to include any additional environment viarables.